app/controllers/songs_controller.rb

```diff
@@ -10,6 +10,10 @@ def upload
     @song = Song.create(params[:song])
     @song.user_id = current_user.id
     if @song.save
+      binding.pry
+      current_user.followed_users.each do |follower|
```

Comment:
As was mentioned before, let's move all Mail stuff to separate module and include it in appropriate places